ci-operator/step-registry/cucushift/upgrade/check-not-recommended/cucushift-upgrade-check-not-recommended-commands.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+# Extract oc binary which is supposed to be identical with target release
+function extract_oc(){
+    echo -e "Extracting oc\n"
+    local retry=5 tmp_oc="/tmp/client-2"
+    mkdir -p ${tmp_oc}
+    while ! (env "NO_PROXY=*" "no_proxy=*" oc adm release extract -a "${CLUSTER_PROFILE_DIR}/pull-secret" --command=oc --to=${tmp_oc} ${TARGET});
+    do
+        echo >&2 "Failed to extract oc binary, retry..."
+        (( retry -= 1 ))
+        if (( retry < 0 )); then return 1; fi
+        sleep 60
+    done
+    mv ${tmp_oc}/oc ${OC_DIR} -f
+    which oc
+    oc version --client
+    return 0
+}
+
+if [[ -f "${SHARED_DIR}/kubeconfig" ]] ; then
+    export KUBECONFIG=${SHARED_DIR}/kubeconfig
+fi
+
+# Setup proxy if it's present in the shared dir
+if [[ -f "${SHARED_DIR}/proxy-conf.sh" ]]; then
+    # shellcheck disable=SC1091
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+# Get the target upgrades release, by default, OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE is the target release
+# If it's serial upgrades then override-upgrade file will store the release and overrides OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE
+# upgrade-edge file expects a comma separated releases list like target_release1,target_release2,...
+export TARGET_RELEASES=("${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}")
```

Comment:
I am tidying up the similar code in https://github.com/openshift/release/pull/50219/files#diff-a71e3444f4cc5e3d7ecb8ebd36fa8bd1fda181a0806d09523b318eadffaebc71L756-L765, so I think you also can do the same thing here to keep consistent. I think you can directly drop these lines, and use `export TARGET="${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}"` on line 54.